### ISSUE TITLE
Deprecate/remove old repos

### ISSUE THREAD

**Issue by Akarshit** [reactioncommerce/reaction-platform#178]
This is the proposal for deprecating and archiving some of the repos. The repos that are deprecated would get no updated 
(except security updates). This would really help us to reduce the surface area of the things we are maintaining.
*(Many of these are private repos.)*
## Deprecating Repos
1. https://github.com/reactioncommerce/catalyst - We don't maintain this style system anymore
2. https://github.com/reactioncommerce/reaction-component-library - We don't maintain these components system anymore
3. https://github.com/reactioncommerce/hydra-token - Would not be needed after we migrate off hydra.

## Archiving Repos
- [x] https://github.com/reactioncommerce/micro-frontend-admin - No longer on roadmap
- [x] https://github.com/reactioncommerce/admin-plugin-products - Was used by micro-frontend-admin
- [x] https://github.com/reactioncommerce/generator-reaction - No longer needed.
- [x] https://github.com/reactioncommerce/reaction-devtools - Not maintained.
- [x] https://github.com/reactioncommerce/reaction-health-check - A small plugin that's not really needed.
- [x] https://github.com/reactioncommerce/mongo-s3-backup - We don't need this for any purpose.
- [x] https://github.com/reactioncommerce/admin-plugin-example - No longer on road-map
- [x] https://github.com/reactioncommerce/reaction-static - The site is gone now, so we don't need this.
- [x] https://github.com/reactioncommerce/reaction-blog - The blog is also gone now.
- [x] https://github.com/reactioncommerce/api-plugin-pricing-engine - Was part of enterprise offering which is cancelled.
- [x] https://github.com/reactioncommerce/reaction-static-hugo - No longer needed.
- [x] https://github.com/reactioncommerce/categories - Enterprise offering similar to tags - not used right now, too much to maintain.
- [x] https://github.com/reactioncommerce/pricewatch - Project specific to BOBS, not needed now.
- [x] https://github.com/reactioncommerce/api-plugin-authorization-keto - Looks like some fork of authorization plugin, not sure.
- [x] https://github.com/reactioncommerce/reaction-search - Elasticsearch component, not on the roadmap.
- [x] https://github.com/reactioncommerce/docker-confluent-platform - part of enterprise offering, no longer on roadmap.
- [x] https://github.com/reactioncommerce/pricing-engine - part of enterprise offering, no longer on roadmap.
- [x] https://github.com/reactioncommerce/change-data-capture - part of enterprise offering, no longer on roadmap.
- [x] https://github.com/reactioncommerce/catalog-publisher - part of enterprise offering, no longer on roadmap.
- [x] https://github.com/reactioncommerce/dociql - not sure why we forked this
- [x] https://github.com/reactioncommerce/reaction.categories - part of enterprise offering, no longer on roadmap.
- [x] https://github.com/reactioncommerce/reaction-release-utils - release process repo, just links to notion.
- [x] https://github.com/reactioncommerce/react - well, can't maintain a fork of react for sure.
- [x] https://github.com/reactioncommerce/blessed-contrib - not sure why we forked this.
- [x] https://github.com/reactioncommerce/reaction-graphql-gateway
- [x] https://github.com/reactioncommerce/grafana-import-export - required for migrations of enterprise catalog. Not needed.
- [x] https://github.com/reactioncommerce/reaction-grafonnet - dashboards for migration. Not needed.
- [x] https://github.com/reactioncommerce/logic-engine - no readme, dead project.
- [x] https://github.com/reactioncommerce/reaction-questions - moved to discord.
- [x] https://github.com/reactioncommerce/catalog-publish-deduplicator - part of enterprise offering.
- [x] https://github.com/reactioncommerce/p-truthy-race - not being used in any other project.
- [x] https://github.com/reactioncommerce/node-epimetheus - not sure why we needed this.
- [x] https://github.com/reactioncommerce/docker-jmx-prometheus-exporter - not sure why we needed this.
- [x] https://github.com/reactioncommerce/catalog-prototype - PoC, never extended further.
- [x] https://github.com/reactioncommerce/reaction.cart - Enterprise offering
- [x] https://github.com/reactioncommerce/reaction-cloudformation - cloudformation templates for all the services we no longer use.
- [x] https://github.com/reactioncommerce/reaction-plugin-payment-radial - enterprise offering.
- [x] https://github.com/reactioncommerce/design - discord and individual repos are used to report issues.
- [x] https://github.com/reactioncommerce/reaction-nginx - enterprise offering, not longer needed.
- [x] https://github.com/reactioncommerce/reaction-app-server - doc for installing and starting reaction on EC2. moved to docker.
- [x] https://github.com/reactioncommerce/enzyme - not sure why we needed fork of enzyme.
- [x] https://github.com/reactioncommerce/babel-plugin-inline-import - no longer being used anywhere else.
- [x] https://github.com/reactioncommerce/http-tracking-api - part of enterprise offering.
- [x] https://github.com/reactioncommerce/datagen - used for generating tracking events, no longer needed.
- [x] https://github.com/reactioncommerce/reaction-buildpack - from the meteor era.
- [x] https://github.com/reactioncommerce/example-payments-plugin - WIP since 6 years.
- [x] https://github.com/reactioncommerce/infrastructure-rfc - docs for infrastructure decisions, not used now.
- [x] https://github.com/reactioncommerce/reaction-feature-requests - we use direct issues.
- [x] https://github.com/reactioncommerce/meteor-google-spreadsheets - moved off meteor
- [x] https://github.com/reactioncommerce/meteor-ddp-login - moved off meteor
- [x] https://github.com/reactioncommerce/nodemailer - we have mailchimp now!
- [x] https://github.com/reactioncommerce/transliteration - not used in any other repos.
- [x] https://github.com/reactioncommerce/reaction-export-orders-csv - not a feature we ship by default.
- [x] https://github.com/reactioncommerce/prometheus-operator - fork we no longer use/maintain.
- [x] https://github.com/reactioncommerce/mongodb_exporter - fork we no longer use/maintain.
- [x] https://github.com/reactioncommerce/charts -  fork we no longer use/maintain.
- [x] https://github.com/reactioncommerce/infrascripts - scripts for infrastructure stuff. Not needed now.
- [x] https://github.com/reactioncommerce/prometheus-operator-private - used to install prometheus, which is not needed now.
- [x] https://github.com/reactioncommerce/helmcharts - Reaction on Kube, not on the roadmap.
- [x] https://github.com/reactioncommerce/kops-tooling - Kube and other advanced stuff we don't use.
- [x] https://github.com/reactioncommerce/chart-gitlab - dead
- [x] https://github.com/reactioncommerce/kube-helm - dead
## Undecided Repos
1. https://github.com/reactioncommerce/data-factory
1. https://github.com/reactioncommerce/reaction-automation-qa
1. https://github.com/reactioncommerce/reaction-catalog-sample-data
1. https://github.com/reactioncommerce/reaction-plugin-routing
2. https://github.com/reactioncommerce/healthcheck
3. https://github.com/reactioncommerce/reaction-data-loader
4. https://github.com/reactioncommerce/proxy-traefik - Still relevant but will be deprecated soon.

Would love to get some feedback on the list.

**Comment by kieckhafer** [comment]
A quick scan of the list and the only one that sticks out to me is we might want to keep #3 from your undecided repos (https://github.com/reactioncommerce/reaction-catalog-sample-data).

**Comment by pankajpatel** [comment]
I was using https://github.com/reactioncommerce/proxy-traefik as a base to learn and make a production ready environment 

Can you put more light on why this won't be needed anymore?

**Comment by Akarshit** [comment]
@pankajpatel The idea was that we will try to find a simpler method to help people deploy Reaction. But your comment gives valuable information (that the repo is still relevant).
I will definitely move it from archiving to deprecate. We might not have the manpower to maintain it, but can definitely keep it open for community use.

**Comment by pankajpatel** [comment]
@Akarshit Thank you for keeping it there for community to learn about the strategy.

I would be happy to see some leaner/simpler approach to deploy Reaction. 

This repo allowed me to learn/experiment with Ansible and Traefik for Reaction.

I have a (almost) live deployment from this repo and I still don't have confidence to expose the site out in public. 
But this repo is the best I could find to have close-to-ProductionReady Reaction.

**Comment by Akarshit** [comment]
I have updated the list with a line of reason as to why it's being archived/deprecated. 
I will start taking action on this list next week.

**Comment by Akarshit** [comment]
I have archived a bunch of repos, waiting on more admin permission and will archive the rest then.

**Comment by delagroove** [comment]
Thanks for the comments, will continue with the repos that were recommended and archive the others
